packages/contracts-bedrock/src/cannon/MIPS.sol

```diff
@@ -730,7 +667,12 @@ contract MIPS is ISemver {
             }
 
             if ((opcode >= 4 && opcode < 8) || opcode == 1) {
-                return handleBranch(opcode, insn, rtReg, rs);
+                st.CpuScalars memory cpu = getCpuScalars(state);
+
+                ins.handleBranch(cpu, state.registers, opcode, insn, rtReg, rs);
```

Comment:
```suggestion
                ins.handleBranch({_cpu: cpu, _registers: state.registers, _opcode: opcode, _insn: insn, _rtReg: rtReg, _rs: rs});
```

nit: Functions that require alot of args should be called with named parameters.